api/src/main/java/com/inrupt/client/ClientHttpException.java

```diff
@@ -0,0 +1,53 @@
+package com.inrupt.client;
+
+import com.inrupt.client.spi.JsonService;
```

Comment:
The API class should not consume the SPI. It makes a circular dependency in the package graph

Reply:
Fixed with 8f17f98e98 (and more generally with 85b25ebb8d399aabad4bfeca1ca00a28bad54675)